packages/default/scss/button/_layout.scss

```diff
@@ -25,11 +24,15 @@
 
         // TODO: extend disabled selector
         &[disabled],
-        &.k-state-disabled:hover,
-        &.k-state-disabled:focus,
         [disabled] & {
             @extend %disabled;
         }
+        .k-ie9 &.k-state-disabled:hover,
+        .k-ie9 &.k-state-disabled:focus,
+        .k-ie10 &.k-state-disabled:hover,
+        .k-ie10 &.k-state-disabled:focus {
+            @extend %disabled;
+        }
```

Comment:
I split this, because IE 9 and 10 do not support `pointer-events`. I guess we could rewrite it with `.k-no-pointer-events &.kstate-disabled...` at some point.